app/[locale]/layout.tsx

```diff
@@ -49,8 +49,6 @@ export const generateStaticParams = () => {
   });
 };
 
-export const dynamicParams = false;
```

Comment:
`dynamicParams = false` restricts routing to only the statically generated params. After the homepage migration this was `en` and `es` which worked fine. Now we have a mix of statically generated + dynamically rendered pages. It is not documented, but setting `dynamicParams = false` sets this for all child routes too which would cause pages like the news posts to 404.